.github/workflows/build-images.yaml

```diff
@@ -30,14 +30,25 @@ jobs:
         id: add-branch-tag
         run: |
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Set Authorino version
-        id: authorino-version
+      - name: Set Authorino git sha
+        id: authorino-git-sha
         run: |
           if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
-          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+          echo "version=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
           else
-          echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
+          echo "git_sha=${{ github.sha }}" >> $GITHUB_ENV
```

Comment:
When it's a release tag, we do not set the Git SHA?

Reply:
I can ensure it happens in all scenarios ?